scripts/automation/Radius/readme.md

```diff
@@ -144,6 +144,22 @@ Output for re-running commands:
 ### End User Experience
 
 After a user's certificate has been distributed to a system, those users can then connect to a radius network with certificate based authentication.
+### Importing a certificate
+
+If you have your own self-signed certificate, you can import the certificate and key to `~/Radius/Cert` folder. Note: Please make sure the certificate and key name ends with `key.pem` and `cert.pem` (ex. `radius_ca_cert.pem` or `radius_ca_key.pem`)
+
+### Importing a certificate with a .pfx file
+
+In order to import a certificate from a .pfx file, the certificate and key needs to be extracted from .pfx file.
+
+1. Extract the private key: `pkcs12 -in certfile.pfx -nocerts -out ~/Radius/radius_ca_key.pem -nodes`
```

Comment:
```suggestion
1. Extract the private key: `openssl pkcs12 -in certfile.pfx -nocerts -out /projectDir/Radius/Cert/radius_ca_key.pem`
```

Reply:
Missing 'openssl' and when I tested with -nodes I found that it stripped my original password set on the key file. [According to the documentation](https://www.openssl.org/docs/man3.0/man1/openssl-pkcs12.html) it's an option to not encrypt the private key, so I suggest we remove it.